lib/index.js

```diff
@@ -188,26 +175,15 @@ internals.docs = function (settings, server) {
 };
 
 
-internals.getConnectionsData = function (connections) {
-
-    connections.forEach((connection) => {
-
-        connection.table = internals.getRoutesData(connection.table);
-    });
-
-    return connections;
-};
-
-
-internals.getRoutesData = function (routes) {
+internals.getRoutesData = (routes) => {
 
     return routes.map((route) => ({
         path: route.path,
         method: route.method.toUpperCase(),
         description: route.settings.description,
         notes: internals.processNotes(route.settings.notes),
         tags: route.settings.tags,
-        auth: internals.processAuth(route),
+        auth: route.settings.auth,
```

Comment:
I think this removed the lookup that happened in the previous version, was something wrong with that ?

Reply:
I assumed we were dropping support for anything under Hapi 17. Meaning that code should be redundant ✌️

Reply:
The old function was not purely old hapi, only the if part was.

Reply:
Maybe I'm totally missing something but the functions simply returns `route.settings.auth` without the if part. So not sure what it serves without it ✌️

Reply:
```js
internals.processAuth = (route) => {

    const auth = route.settings.auth;

    /* $lab:coverage:off$ */
    if (auth && (auth.entity || auth.scope)) { // Hapi < 12
        auth.access = [{
            entity: auth.entity,
            scope: {
                selection: auth.scope
            }
        }];
    }
    /* $lab:coverage:on$ */

    return auth;
};
```

Reply:
I'll add it back regardless

Reply:
Except it's not, look at the diff, it returned `route.connection.auth.lookup(route)`.

Reply:
Oh damn, my bad. Totally forgot I did that in my old commits. Good catch ✌️

Reply:
I dug a little deeper and `connection` was for the multi connection that is gone now and `lookup()` was a Hapi function that was related to that and removed in v17. So it's definitely correct to remove it. https://hapijs.com/api/16.6.2#serverlookupid

Reply:
I updated it to work with Hapi 17, since `connection.auth.lookup(request.route)` got replaced with `server.auth.lookup(request.route)`

https://hapijs.com/api/16.6.2#serverauthdefaultoptions
https://hapijs.com/api/17.2.0#-serverauthdefaultoptions